src/edge/tests/test_io.py

```diff
@@ -45,18 +44,39 @@ def test_outputs_gff(self):
 
         # be aware of the tabs in the string below
         expected = """##gff-version 3
-##sequence-region Bar 1 19
-Bar\tfeature\tgene\t2\t2\t.\t+\t.\tname=A1
-Bar\tfeature\tgene\t9\t15\t.\t+\t.\tname=A1
+##sequence-region Bar 1 13
+Bar\tfeature\tgene\t2\t9\t.\t+\t.\tname=A1
 ##FASTA
 >Bar <unknown description>
 %s
 """ % (fragment.sequence,)
         self.assertEquals(expected, gff)
 
-    def test_outputs_gff_handles_negative_strand(self):
+    def test_can_output_gff_feature_on_reverse_strand(self):
         fragment = self.fragment.indexed_fragment()
         fragment.annotate(2, 9, 'A1', 'gene', -1)
+
+        with tempfile.NamedTemporaryFile(mode='w+', delete=False) as f:
+            f.close()
+            IO(self.genome).to_gff(f.name)
+            h = open(f.name, 'r')
```

Comment:
what does h mean here? can we use a more descriptive name than h?

Reply:
I'm ok with f being single letter variable since it's more common and easy to understand.

Reply:
h = handle. But I can change it.

Reply:
Thanks! This looks much better!